### ISSUE TITLE
diarization not working

### ISSUE THREAD

**Issue by lifengmds** [QuentinFuxa/whisper_streaming_web#141]
Hi, thank you for your work, I've been play around this lib for a month or two now. 

The transcription works fine, but the speaker diarization is not working on Chinese language , it could not even recognize voices of man and woman , it does not recognize most of the time, or wrongly recognized some sentences.

Model: faster-whisper custom model on huggingface: [XA9/Belle-faster-whisper-large-v3-zh-punct-int8](https://huggingface.co/XA9/Belle-faster-whisper-large-v3-zh-punct-int8)

According to pyannote diarization model description, it has pre-trained on Chinese audio data collection: AliMeeting   https://huggingface.co/pyannote/speaker-diarization-3.1

Another issue is that it hallucinate a little bit , I wonder if it's because of the whisper model.

**Comment by lifengmds** [comment]
I suggest to add a new option to use pretained and fine-tuned pyannote diarization models, such as models listed here https://huggingface.co/diarizers-community

**Comment by CQQ-biu** [comment]
hello 您好，我最近也在测试这个项目，发现在中文说话人识别上确实效果一般，请问您有什么优化的方案吗？我尝试了很多都没有效果，打扰了

**Comment by lifengmds** [comment]
> hello 您好，我最近也在测试这个项目，发现在中文说话人识别上确实效果一般，请问您有什么优化的方案吗？我尝试了很多都没有效果，打扰了

the model accuracy has nothing to do with this project. you have to figure it out yourself.

**Comment by lifengmds** [comment]
I was asking the wrong question.
